### ISSUE TITLE
Sequential disk backup

### ISSUE THREAD

**Issue by pboguslawski** [ganeti/ganeti#1630]
gnt-backup (checked in Debian 9 and Debian 11) exports/imports all VM disks in parallel which may overload hardware in case of many disks in VM. Disks should be processed sequentially.

**Comment by pboguslawski** [comment]
Trying to backup guest with 15 vitual disks even on HW RAID with 10k disks throws 60s timeouts; not all disk sending and/or writing processes are started in 60s; workaround is to change DISK_TRANSFER_CONNECT_TIMEOUT 60 to something like 300. Would be nice to have it configurable in cluster params to avoid ugly patching.

Backup processes are executed with same prio in system like guests; nice -n 19 would be better probably.
